### PR TITLE
[Refactor] Fix Makefile fmt target

### PR DESCRIPTION
**Pull request by bquenin**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -223,9 +223,11 @@ fmt:
 	@goimports -l $(CHECKDIRS) && goimports -w $(CHECKDIRS)
 	@gofmt -s -l -w $(CHECKSRCS)
 
+# TODO: Remove the cli folder exclusion when we're done
 .PHONY: lint
 lint:
 	@gometalinter --deadline=10m --concurrency=1 --enable-gc --vendor --exclude=vendor --exclude=\.pb\.go \
+	    --exclude=cli \
```
Comment (subfuzion) [inline Makefile:230]: Why are we excluding `cli`?

Comment (bquenin) [reply Makefile:230]: Because it has lots of linting issues and since it's still moving parts. I thought I'll deal with it with another PR.

Comment (subfuzion) [reply Makefile:230]: I'm glad that you actually caught the omission in the search criteria, but I don't feel comfortable with ignoring lint checks; we should be adding code as we go that meets our quality criteria ... I know we're trying to get all the account code merged in quickly, but how long do you anticipate a need to ignore lint issues?

Comment (bquenin) [reply Makefile:230]: I can do it tomorrow

**Review by subfuzion — CHANGES_REQUESTED** [review]
Please address [build failure](https://travis-ci.org/appcelerator/amp/builds/219512596).

File: Makefile (hunk 1 above)
Comment (subfuzion) [reply Makefile:230]: 👍

**Review by subfuzion — APPROVED** [review]
👍

**Pull request merged**